homeassistant/components/toon/sensor.py

```diff
@@ -68,6 +69,20 @@ async def async_setup_entry(
                 )
             ]
         )
+        
+    if coordinator.data.water_usage:
```

Comment:
This check isn't needed, it will always have that.

Reply:
Even if there's no water sensor connected?

Reply:
Yes, as the data is always provided by the API, there is no way of telling it is active or not.